parsing/CSVFileParser/HarlockOfficial_csv_file_parser/linked_list.c

```diff
@@ -0,0 +1,56 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "linked_list.h"
+
+struct node_t{
+    void* val;
+    struct node_t *previous;
+    struct node_t *next;
+};
+
+void init_list(linked_list *list){
+    (*list).head = NULL;
```

Comment:
Why have you written `(*list).head` instead of `list->head`?

Reply:
didnt use c in years, also did the software without ide, fixing it